static/app/views/performance/newTraceDetails/traceApi/useTrace.tsx

```diff
@@ -58,7 +58,7 @@ export function getTraceQueryParams(
     limit = parseInt(limit, 10);
   }
 
-  const eventId = decodeScalar(normalizedParams.eventId);
+  const targetId = decodeScalar(normalizedParams.targetId ?? normalizedParams.eventId);
```

Comment:
Some people may have old links that this will break. 

Lets keep eventId ad fallback in case targetId is not set

Reply:
https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_events_trace.py#L1069-L1071

The targetId is treated as a span id first and falls back to an event id the way it used to.

Reply:
Yep @JonasBa, old links will work.